docs/developers_guide/micro_frontends_in_open_edx.rst

```diff
@@ -16,35 +16,168 @@ To run an MFE locally you must install `node 12 <https://nodejs.org>`_ (with npm
   npm run test
   npm run build (build static output)
 
+****************************************
 Configuration with Environment Variables
-----------------------------------------
+****************************************
 
-Open edX MFEs expect a number of process environment variables (Accessible in node via ``process.env``) in order to properly configure authentication, routing to other parts of the system, etc. Each MFE has ``.env``, ``.env.development``, and ``.env.test`` files that are consumed by the MFE. The production .env file contains only null values – the build process should supply them via the command line prior to running ``npm run build``.
+Open edX MFEs expect a number of process environment variables (Accessible in node via ``process.env``) in order to properly configure the application. This includes links to other parts of the system, branding elements like the site name and logos, cookie names, and authentication configuration details. Each MFE has ``.env``, ``.env.development``, and ``.env.test`` files that are consumed by the MFE. The production ``.env`` file contains only falsy values – the build process should supply them via the command line prefixing the ``npm run build`` command.
 
-Example ``.env.development``:
+Production build example::
 
-.. code-block:: bash
+  SITE_NAME="My Site Name" npm run build
+
+The above would only set the SITE_NAME - suitable values for the environment variables should be supplied as well.
+
+Required Environment Variables
+==============================
+
+All environment variables in this section are required for Open edX micro-frontends.
+
+A note on micro-frontend-specific environment variables
+-------------------------------------------------------
+
+Individual micro-frontends may depend on other environment variables not on this list.  Those variables will be documented in the README files in individual micro-frontend repositories, but can be specified on the command line in the same way as described above.
+
+Fundamentals
+------------
+
+``NODE_ENV``
+  Example: ``production`` | ``development`` | ``test``
+
+  Used by both the build process and application to determine what type of environment we're running. Valid values are "development", "test", and "production". All production-like environments should use "production", as this removes development warnings and checks, and can enable code optimizations for many dependencies like React.
+
+``PUBLIC_PATH``
+  Example: ``/``
+  Used by both the build process and application to set the path of the application.  Should include an initial forward slash ``/``.  Will default to ``/`` if not specified.
+
+``BASE_URL``
+  Example: ``https://new-mfe.example.com``
+
+  The fully-qualified URL to the micro-frontend being built. Used by frontend-platform as part of authentication, as well as by many applications for internal links.
```

Comment:
should this always have to be a subdomain of your domain?

Reply:
It doesn't need to be, adjusting to include that option.